Fix Copilot Code Review incorrectly flagging PROTOTYPE comments in feature branches (#83555)
dotnet/roslyn: .github/skills/code-review/SKILL.md

- **Track deferred work with GitHub issues.** No `// TODO` comment should be added or merged. Instead use a comment like `// Tracked by <URL>: description` to link to a GitHub issue tracking the follow-up work.

> - **PROTOTYPE comments are allowed in feature branches.** Do not flag `// PROTOTYPE` comments when the PR targets a feature branch (any branch other than `main`). PROTOTYPE comments are intentionally used to mark temporary or experimental code in feature branches and are enforced by CI only when merging to `main`. If a PR targets `main`, PROTOTYPE comments must be removed or replaced before merging.

Reviewer: Keep this short. "**PROTOTYPE comments are allowed in feature branches.** CI only prevents PROTOTYPE comments from being merged into `main`"

> - **PROTOTYPE comments are allowed in feature branches.** Do not flag `// PROTOTYPE` comments when the PR targets a feature branch (any branch other than `main`). PROTOTYPE comments are intentionally used to mark temporary or experimental code in feature branches and are enforced by CI only when merging to `main`. If a PR targets `main`, PROTOTYPE comments must be removed or replaced before merging.

Reviewer: ```suggestion
- **PROTOTYPE comments are allowed in feature branches.** CI only prevents PROTOTYPE comments from being merged into `main`
```
